challenge-3/src/findings.ts

```diff
@@ -0,0 +1,34 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+import { CHAIN_IDS } from "./constants";
+
+export const createEscrowFinding = (balanceArbitrum: string, balanceOptimism: string): Finding => {
+  return Finding.fromObject({
+    name: "MakerDAO's L1 escrow balance of Arbitrum & Optimism",
+    description: `Arbitrum's escrow balance ${balanceArbitrum} - Optimism's escrow balance ${balanceOptimism}`,
+    alertId: "L1-Escrow-Balance",
```

Comment:
Very small change but I believe alertIds should be all capitalized, same for line 24. If you change it, make sure to also update it in your agent.ts. Could create constants for your alertIds.